app/assets/src/components/common/Heatmap/MetadataSelector.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
```

Comment:
Same with this component. Add a comment explaining what it does and that it's used in both heatmaps.